torch/export/_trace.py

```diff
@@ -928,6 +937,12 @@ def _export(
             f"Expecting `args` to be a tuple of example positional inputs, got {type(args)}",
         )
 
+    if _disable_forced_specializations and strict:
+        raise UserError(
+            UserErrorType.INVALID_INPUT,
+            "_disable_forced_specializations can be only be specified in non-strict mode.",
```

Comment:
Why?

Reply:
It's really messy to try to pass the flag into Dynamo's call to produce_guards(): https://github.com/pytorch/pytorch/blob/9aa7699185e4ec39077e3046dfd63244dffa9ddb/torch/_dynamo/guards.py#L1447